pkg/config/config_test.go

```diff
@@ -37,3 +40,96 @@ func TestDefaults(t *testing.T) {
 		t.Fatalf("Defaults should match but didn't")
 	}
 }
+
+func TestMerge(t *testing.T) {
```

Comment:
these tests seem pretty much identical to the ones above. Are they exercising something significantly different?

Reply:
true, i'll clean these up